src/pages/Shopping/shoppingList.js

```diff
@@ -1,37 +1,67 @@
 import React, { Component } from 'react';
-import Item from './Item/Item';
+import Category from './Category/Category';
+import Item from '../../components/Item/Item';
 
-import './shoppingList.scss';
+import './ShoppingList.scss';
+
+const CATEGORY_LIST = [
+  {
+    id: 1,
+    title: 'allmeat',
+    name: '전체',
+  },
+  {
+    id: 2,
+    title: 'pig',
+    name: '돼지',
+  },
+
+  {
+    id: 3,
+    title: 'beef',
+    name: '소',
+  },
+  {
+    id: 4,
+    title: 'chicken',
+    name: '닭',
+  },
+  {
+    id: 5,
+    title: 'susan',
+    name: '수산',
+  },
+];
+
+class ShoppingList extends Component {
+  state = {
+    items: [],
+  };
+
+  componentDidMount() {
+    fetch('/data/itemList.json', {
+      headers: {
+        method: 'GET',
+      },
+    })
+      .then(res => res.json())
+      .then(data => this.setState({ items: data }));
```

Comment:
- headers 는 method 프로퍼티를 가지지 않습니다.
- 아마 `{ method: 'GET', headers: { ...헤더내용 } }` 이렇게 작성하시는 걸 헷갈리신 걸까요?
- 그리고 fetch 함수는 기본적으로 get 메서드이기 때문에 get 메서드의 경우 옵션을 생략해줄 수 있습니다.
```suggestion
    fetch('/data/itemList.json')
      .then(res => res.json())
      .then(data => this.setState({ items: data }));
```